packages/@n8n/db/src/entities/credentials-entity.ts

```diff
@@ -42,6 +42,25 @@ export class CredentialsEntity extends WithTimestampsAndStringId implements ICre
 	@Column({ default: false })
 	isGlobal: boolean;
 
+	/**
+	 * Whether the credential can be dynamically resolved by a resolver.
+	 */
+	@Column({ default: false })
+	isResolvable: boolean;
```

Comment:
Not familiar with the details of this project, can we rely on `resolverId` being `null` or not `null` for this?

Reply:
actually no, because the resolverId can be null, and isResolvable true. It's when the resolver has been setup for the workflow instead of the credentials